ui/DesignSystem/Component/SourceBrowser/File.svelte

```diff
@@ -1,8 +1,20 @@
 <script>
+  import { link } from "svelte-spa-router";
```

Comment:
Not for this PR but for a future improvement, we should have a `Link` component which hides away that we use this specific package for that concern.